lib/wprof/reporters/file_report.rb

```diff
@@ -0,0 +1,63 @@
+module Wprof
+  module Reporters
+    module FileReport
+      def file_report
+        require 'csv'
+        csv_type = WProf::Config.get_value(:csv_type)
+        path = WProf::Config.get_value(:file_path)
+        if csv_type == 'MIX'
```

Comment:
I think we can use a little of metaprogramming here to make the code easier by using dynamic dispatch, instead of the if/else clause we can:
```
  send('write_#{csv_type.undercase}_file',path)
```

So the whole method could be:
```
def file_report
  require 'csv'
  csv_type = WProf::Config.get_value(:csv_type)
  path = WProf::Config.get_value(:file_path)
  send('write_#{csv_type.undercase}_file',path)
end
```

Reply:
Plus! If we add another way to write a file report, we don't need to change this method 😉

Reply:
Looks good for this case!.

Reply:
done